components/webstories/Stories.tsx

```diff
@@ -227,10 +210,9 @@ const Stories = ({
           className="flex flex-col h-full w-full md:rounded-xl lg:rounded-xl xl:rounded-xl"
         />
       )}
-
       {currentIndex < totalLen - 1 && (
         <div className="absolute w-full z-30 bottom-0 animate-grow cursor-text">
-          <div className="bg-secondary-300 opacity-80 p-8 md:rounded-b-xl md:rounded-t-sm lg:rounded-b-xl lg:rounded-t-sm xl:rounded-t-sm xl:rounded-b-xl">
+          <div className="bg-secondary-300 opacity-80 p-8 md:rounded-b-xl lg:rounded-b-xl xl:rounded-b-xl">
```

Comment:
why you removed all the top border class can replace with one

Reply:
Reverted